app/settings/globals.py

```diff
@@ -81,6 +81,10 @@
 WRITER_PORT: int = config("DB_PORT", cast=int, default=DB_WRITER_SECRET["port"])
 WRITER_DBNAME = config("DATABASE", cast=str, default=DB_WRITER_SECRET["dbname"])
 
+if ENV == "dev":
+    NAME_SUFFIX = config("NAME_SUFFIX", cast=str)
+    READER_DBNAME = f"{READER_DBNAME}{NAME_SUFFIX.replace('-', '_')}"
```

Comment:
This replace is done a couple times, could you just pass in the NAME_SUFFIX from terraform with it already replaced?